package.json

```diff
@@ -41,6 +41,7 @@
     "babel-preset-env": "^1.6.1",
     "babelify": "^8.0.0",
     "browserify": "^15.1.0",
+    "eslint": "^4.15.0",
```

Comment:
I added this to the dev dependencies because PyCharm seems to want it to be in the node_modules directory in order for ESLINTing to work in the IDE.